thangs_fetcher.py

```diff
@@ -736,12 +554,12 @@ def get_stl_search(self, stl_path):
             self.selectionSearching = False
             self.searching = False
             self.newSearch = False
-            self.SelectionFailed = True
+            self.selectionFailed = True
             return
 
         # if os.path.isfile(stl_path):
         #    os.remove(stl_path)
-        #self.Thangs_Utils.clean_downloaded_model_dir("ThangsSelectionSearch")
+        # self.Thangs_Utils.clean_downloaded_model_dir("ThangsSelectionSearch")

```

Comment:
Delete more commented out code.

Reply:
This is part of the clean up of get_stl_search function that @RandyHucker is doing next.